SQL/New_patches/2019-11-27-RemoveOldConfigSettings.sql

```diff
@@ -0,0 +1,4 @@
+DELETE FROM ConfigSettings WHERE Name='Logs';
+DELETE FROM ConfigSettings WHERE Name='css';
+DELETE FROM ConfigSettings WHERE Name='issue_tracker_url';
+DELETE FROM ConfigSettings WHERE Name='FileGroup';
```

Comment:
This should go into cleanup patches, and wasn't there some controversy about `issue_tracker_url`?

Reply:
From what I recall there was meant to be a roadmap meeting where you and Rida (and Samir?) were going to discuss what to do about it.

Reply:
Sorry, I don't think that ever happened. I'll try and remember to bring it up next time.

Reply:
Thanks!